src/core/mesh/qgsmeshtimesettings.h

```diff
@@ -57,6 +57,13 @@ class CORE_EXPORT QgsMeshTimeSettings
     //! Sets number of offset hours for relative time formatting
     void setRelativeTimeOffsetHours( double relativeTimeOffsetHours );
 
+    //! Returns number of seconds used as interval for dataset playback
+    //! \since QGIS 3.12
```

Comment:
looks like it is not picked by SIP, it is not in python docs... maybe use /** */ doxygen instead of //!... or maybe you forgot to run sipyfy?